value-propositions.md

```diff
@@ -1,22 +1,25 @@
+Key material management and access condition verification are operationally distributed across a diverse array of machines/servers, run by economically independent individuals and commercial entities.
 
-**Powerful, per-ciphertext conditionality**\
-Future access to data can be made contingent on the fulfillment of predefined conditions, and those conditions attached to any granularity of data payload – a single message, an entire table, or a petabyte of video footage.&#x20;
+**Powerful, per-payload conditionality**\
+Future access to data can be made contingent on the fulfillment of predefined conditions, and those conditions attached to any granularity of data payload – a single message, an entire table, or a petabyte of video footage.
 
 **Flexible condition composability**\
-Conditions of all types – NFT-holding, RPC, time-based, known keypairs – can be mixed-and-matched using logical operators and flexible prefix notation into virtually any desired combination. Conditions can also be flexibly surfaced at different stages of runtime.&#x20;
+Conditions of all types – NFT-holding, RPC, time-based, known keypairs – can be mixed-and-matched using logical operators and flexible prefix notation into virtually any desired combination. 
+Conditions can also be flexibly surfaced at different stages of runtime.
 
 **Full control over access managers & network parameters**\
-Developers have total and perpetual control over the cohort(s) of nodes which manage access to a given data payload, user segment or entire application. Network parameters, such as the size or composition of the cohort, can be tuned directly by the developer, or packaged into user-facing optionality to accommodate diverse risk preferences.\
-_Note: parameter tuning will be available in future versions of TACo._&#x20;
+Network parameters, such as the size or composition of the cohort of nodes which manage access to a given data payload, can be tuned directly by the developer, or packaged into user-facing optionality to accommodate diverse risk preferences.
+_Note: parameter tuning will be available in future versions of TACo._
 
 **Threshold's track record for uptime, reliability and competence**\
-The Threshold network's multi-app model incentivizes all TACo nodes to simultaneously provision service to tBTC, and abide by its strict availability requirements. With over [$130m](https://dune.com/threshold/tbtc) in TVL protected by Threshold nodes, TACo 'piggybacks' on tBTC's uptime, reliability, technical competence, and security track record.&#x20;
+The Threshold network's multi-app model incentivizes all TACo nodes to simultaneously provision service to tBTC, and abide by its strict availability requirements.
+With over [$200m](https://dune.com/threshold/tbtc) in TVL protected by Threshold nodes, TACo 'piggybacks' on tBTC's uptime, reliability, technical competence, and security track record.
```

Comment:
Should we make this more evergreen so we don't need to update the number periodically?

For example:
`With the ("substantial" | "considerable" | "massive" ) amount in TVL ...`
OR ...